src/helpers.php

```diff
@@ -192,3 +200,89 @@ function progress(string $label, iterable|int $steps, ?Closure $callback = null,
 
     return $progress;
 }
+
+/**
+ * Multi-byte version of wordwrap.
+ *
+ * @param  non-empty-string  $break
+ */
+function mb_wordwrap(
```

Comment:
Wondering whether it's better to encompass this in a method on the `TextareaPrompt` class to prevent it showing up in IDE completion.